### PR TITLE
Make 4-argument select() handle UTF8-flagged strings correctly.

### PR DESCRIPTION
**Pull request by FGasper**
Issue #19882

### CODE REVIEW

File: pod/perldelta.pod [hunk 1]
```diff
@@ -363,7 +363,7 @@ files in F<ext/> and F<lib/> are best summarized in L</Modules and Pragmata>.
 
 =item *
 
-XXX
+4-argument select now properly handles UTF8-flagged strings.
```
Comment (leonerd) [inline pod/perldelta.pod]: C<select()>

Also, this doesn't really say what it does with those strings. What is the correct semantics here? I'm not sure offhand I can imagine what "properly" means in this case

File: t/op/sselect.t [hunk 2]
```diff
@@ -106,16 +106,21 @@ select ($tie, undef, undef, $tie);
 ok("no crash from select $numeric_tie, undef, undef, $numeric_tie");
 
 SKIP: {
-    require POSIX;
+    my $SKIP_CR = sub {
+        skip shift, 4;
+    };
+
+    eval 'require POSIX' or do {
```
Comment (Leont) [inline t/op/sselect.t]: Why quotes and not braces?

Comment (FGasper) [reply t/op/sselect.t]: No good reason. I think I thought the braces form caused a broken test, but that’s probably a mistake. Updated.

File: pod/perldelta.pod (hunk 1 above)
Comment (tonycoz) [reply pod/perldelta.pod]: I think this is resolved with the new perldelta message.

**Comment by FGasper** [comment]
Conflict fixed.

**Pull request merged**